src/Application.tsx

```diff
@@ -193,7 +193,7 @@ export const Application = connector(
               >
                 <Tab
                   value={0}
-                  icon={<img src={PSEIcons.Dataset} />}
+                  icon={<SvgIcon style={{ fontSize: 64 }} viewBox="0 0 18.521 18.521" component={PSEIcons.Dataset} />}
```

Comment:
Could you create a component for that? fontSize and viewBox do not change only the component, which could be passed as a prop

Reply:
done